calcom/cal.com
# feat: Time blocks

## What does this PR do?

Please note that the time blocks currently only work for Google and Office-365 Calendars.



- Fixes #13305 (GitHub issue number)
- Fixes CAL-2962 (Linear issue number - should be visible at the bottom of the GitHub issue description)


https://www.loom.com/share/135020a75791426a8958e6429e55e5fb?sid=75b9446c-73aa-4546-8f3a-855129bf2cb4

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Create event(s) in your calendar with a specific title, say "Roundtable".
- Create a new schedule and add time blocks to that schedule.
- Add the title of the calendar event(s) ("Roundtable") to the time blocks.
- Choose this newly created schedule as the availability schedule for an event type.
- Now visit the booking page for that event type
- Notice that availability is only shown for the exact time of the event(s) in your calendar titled "Roundtable".
